File: app/src/main/java/ru/michanic/mymot/UI/Cells/ReviewsSliderCell.java
package ru.michanic.mymot.UI.Cells;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;

import com.squareup.picasso.Picasso;

import ru.michanic.mymot.Models.YoutubeVideo;
import ru.michanic.mymot.Protocols.Const;
import ru.michanic.mymot.R;

public class ReviewsSliderCell extends RecyclerView.ViewHolder {

    ImageView imageView;

    public ReviewsSliderCell(View itemView) {
        super(itemView);
        imageView = (ImageView) itemView.findViewById(R.id.cell_image);
    }

    public void fillWithVideo(YoutubeVideo video) {
        Picasso.get().load(video.getPreviewPath()).placeholder(R.drawable.ic_placeholder).into(imageView);
    }

}

File: app/src/main/java/ru/michanic/mymot/Kotlin/UI/Cells/ReviewsSliderCell.kt
package ru.michanic.mymot.Kotlin.UI.Cells

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.ImageView
import com.squareup.picasso.Picasso
import ru.michanic.mymot.Models.YoutubeVideo
import ru.michanic.mymot.R

class ReviewsSliderCell(itemView: View) : RecyclerView.ViewHolder(itemView) {
    var imageView: ImageView

    init {
        imageView = itemView.findViewById<View>(R.id.cell_image) as ImageView
    }

    fun fillWithVideo(video: YoutubeVideo) {
        Picasso.get().load(video.previewPath).placeholder(R.drawable.ic_placeholder).into(imageView)
    }
}